### PR TITLE
Jtk7412 branch

### PR DESCRIPTION
**Pull request by JKotowicz**
Added some dummy data that I believe is in the correct format with the new UniversityEmployee name.
Also added in some initial API functions for HR.

### CODE REVIEW

File: API/API.php [hunk 1]
```diff
@@ -431,30 +431,140 @@ function postBook()
 ///////////////////
 
 // Switchboard to Human Resources Functions
-function human_resources_switch($getFunctions)
+function human_resources_switch()
```
Comment (roda0313) [inline API/API.php]: This line shouldn't be changed. I added this to allow everyone to get a list of functions for each team

File: API/API.php [hunk 2]
```diff
@@ -431,30 +431,140 @@ function postBook()
...
 {
 	// Define the possible Human Resources function URLs which the page can be accessed from
-	$possible_function_url = array();
+	$possible_function_url = array("test","updatePerson","upateProf","updateName", "updatePassword");
 
-	if ($getFunctions)
-	{
-		return $possible_function_url;
-	}
-
```
Comment (roda0313) [inline API/API.php]: This also shouldn't be deleted

File: API/API.php [hunk 3]
```diff
@@ -431,30 +431,140 @@ function postBook()
...
+                return updateProfInfo();
+            case "updatePassword":
+                return updatePassword();
+            case "updateName":
+                return updateFullName();
 		}
 	}
-	else
-	{
-		return "Function does not exist.";
```
Comment (ZacharyMoran) [inline API/API.php]: don't remove this please, it lets users know if they tried to access a function that didnt exist

File: API/API.php [hunk 4]
```diff
@@ -431,30 +431,140 @@ function postBook()
...
+			return $exception->getMessage();
+		}
+		logError($exception);
+	}
+	
+	return $success;
+}
+
+function updatePassword() {
+    $username = $_GET["username"];
```
Comment (ZacharyMoran) [inline API/API.php]: you should not be accessing the get params in the function it should be passed in in the switchboard, also anything that updates the db should be using Post not GET

File: API/API.php [hunk 5]
```diff
@@ -431,30 +431,140 @@ function postBook()
...
 	if (isset($_GET["function"]) && in_array($_GET["function"], $possible_function_url))
 	{
 		switch ($_GET["function"])
 		{
-			
+            case "test":
+                return testThis();
+            case "updatePerson":
+                return updatePersonalInfo();
+            case "updateProf":
```
Comment (ZacharyMoran) [inline API/API.php]: Here in the cases is where you need to gather params and check if they even exist, if they dont the function should never be called. Check the StudentEnrollment functions as examples

Comment (JKotowicz) [inline API/API.php]: yeah we will get to that

File: API/API.php (hunk 4 above)
Comment (JKotowicz) [inline API/API.php]: Alright I'll fix that

**Pull request merged**